app/views/devices_guidance/device_specification.md

```diff
@@ -45,22 +47,6 @@ Read more information about <a class="govuk-link" href="/devices/preparing-chrom
 
 [Find the technical specifications for Google Chromebooks](#minimum-specifications) in the table below.
 
-## Apple iPads
-        
-During the ordering process, we’ll ask for your Apple School Manager (ASM) Organisation ID. We recommend you find this ID in advance to avoid any delay in ordering.
-
-It’s a 7 to 8 digit number that can be found by following these steps:
-
-1. Visit <a href="https://school.apple.com" class="govuk-link">https://school.apple.com</a> to sign in to Apple School Manager
-2. Click on ‘Settings’ in the bottom left-hand corner
-3. Click on ‘Enrolment Information’ in the ‘Institution Settings’ section. The Organisation ID (also known as an Apple School Manager ID) will be in the right-hand pane.
```

Comment:
Is this guidance elsewhere, for anyone with problems on devices they might have recently ordered?